test/hotspot/jtreg/compiler/vectorapi/VectorGatherMaskFoldingTest.java

```diff
@@ -224,15 +224,15 @@ public static void testTwoLongVectorLoadGatherMaskedEquals() {
     public static void testTwoLongVectorLoadGatherMaskedNotEqualMask() {
         LongVector res = LongVector.fromArray(L_SPECIES, longArray, 0, longIndices, 0, longVectorMask);
         LongVector res2 = LongVector.fromArray(L_SPECIES, longArray, 0, longIndices, 0, longVectorMask2);
-        Asserts.assertNotEquals(res, res2);
+        Asserts.assertFalse(L_SPECIES.length() != 1 && res.equals(res2));
```

Comment:
You don't need this change either (`longVectorMask` and `longVectorMask` are different even with length 1).